README.md

```diff
@@ -127,8 +149,7 @@ Trigger Chaperone's sign-out process.
 - `signup` - emitted when the user completes a successful sign-up
 - `signout` - emitted when the user competes a successful sign-out
 - `canceled` - emitted when the user purposefully exits sign-in/up
-- `connected` - emitted when the connection is opened
-- `disconnected` - emitted when the connection is closed
-
-> **NOTE:** `signin`, `signup`, and `signout` will never be emitted when the context is
-> `Connection.AUTONOMOUS`.
+- `signal` - emitted when a signal is passed from chaperone
+- `available` - emitted when the connection to chaperone and envoy are opened and hosted app is ready for zome calls
+- `unavailable` - emitted when the ws connection to chaperone and/or envoy is closed
```

Comment:
definitely bikeshedding: On reflection, I think I prefer `ready` and `unready` for these events. NBD at all.

Reply:
My vote would be to keep this for now and follow-up any name changes like these at the end should we still prefer it.  How does that sound?

Reply:
Sounds good to me